livestyled/schemas/user.py

```diff
@@ -104,7 +104,7 @@ class Meta:
     user_emails = RelatedResourceField(schema=UserEmailSchema, data_key='userEmails', many=True)
     user_consent = RelatedResourceField(schema=UserConsentSchema, data_key='userConsent')
     token = fields.String(missing=None)
-    user_aliases = RelatedResourceLinkField(schema=UserAliasSchema, data_key='userAliases', many=True)
+    user_aliases = RelatedResourceField(schema=UserAliasSchema, data_key='userAliases', many=True)
```

Comment:
can you set missing=None?

Reply:
added `missing=[]` at API returns this value